Objective-C/CBLReplicatorConfiguration.h

```diff
@@ -197,46 +158,9 @@ __deprecated_msg("Use -initWithCollections:target: instead.");
  */
 @property (nonatomic) BOOL enableAutoPurge;
 
-/**
- The collections used for the replication.
- */
-@property (nonatomic, readonly) NSArray<CBLCollection*>* collections
-__deprecated_msg("Use collectionConfigs instead.");
-
```

Comment:
I think we should change this to return `NSArray<CBLCollectionConfiguration*>*` to match MultipeerReplicatorConfiguration and deprecate `collectionConfigs`.